frontend/app/routes/_public+/apply+/$id+/federal-provincial-territorial-benefits.tsx

```diff
@@ -163,13 +164,13 @@ export default function AccessToDentalInsuranceQuestion() {
         )}
       </section>
       <div className="flex flex-wrap items-center gap-3">
-        <ButtonLink id="back-button" to="/apply">
+        <ButtonLink type="button" variant="alternative" to={`/apply/${id}/dental-insurance`}>
           <FontAwesomeIcon icon={faChevronLeft} className="me-3 block size-4" />
           {t('dental-benefits.button.back')}
         </ButtonLink>
-        <Button variant="primary" id="continue-button">
+        <Button type="submit" variant="primary">
           {t('dental-benefits.button.continue')}
-          <FontAwesomeIcon icon={faChevronRight} className="ms-3 block size-4" />
+          <FontAwesomeIcon icon={faChevronRight} className="me-3 block size-4" />
```

Comment:
This line shouldn't change.

```suggestion
          <FontAwesomeIcon icon={faChevronRight} className="ms-3 block size-4" />
```